internal/converter/internal/common/river_utils.go

```diff
@@ -61,9 +61,15 @@ func getValueOverrideHook() builder.ValueOverrideHook {
 			return ConvertTargets{
 				Targets: value,
 			}
+		case flow_relabel.Rules:
```

Comment:
Promtail converter code does not use the common utils for this so we need to add code here